lib/codeowners/cli/warner.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module Codeowners
+  module Cli
+    # checks and prints deprecation warnings
```

Comment:
```suggestion
    # Checks and prints deprecation warnings.
```

Reply:
I need to pay more attention to these things!